docs/source/notes/run_name.md

```diff
@@ -53,13 +53,11 @@ See {class}`~.CheckpointSaver` for more information on specifying the arguments
 
 In addition to checkpointing, loggers also use the `run_name` for default logging.
 
-#### Tensorboard Logger
+#### Experiment Tracking Loggers
```

Comment:
Heading changed to `Experiment Tracking Loggers` from `Tensorboard Logger` but the below comment line still refers to old name `TensorboardLogger`. Do we also need to change that?

Reply:
the three comments below it actually refer to the Tensorboard, WandB, and CometML, which all fall under the category of Experiment tracking loggers.